web/core/src/jsMain/kotlin/org/jetbrains/compose/web/css/selectors/CSSSelectors.kt

```diff
@@ -23,13 +23,18 @@ sealed class Nth {
 
 open class CSSSelector {
     override fun equals(other: Any?): Boolean {
-        return this === other || toString() == other.toString()
+        return this === other || asString() == (other as? CSSSelector)?.asString()
     }
 
     internal open fun contains(other: CSSSelector, strict: Boolean = false): Boolean {
         return if (strict) this === other else this == other
     }
 
+    // This method made for workaround because of possible concatenation of `String + CSSSelector`,
+    // so `toString` is called for such operator, but we are calling `asString` for instantiation.
+    // `toString` is reloaded for CSSSelfSelector
+    internal open fun asString(): String = toString()
```

Comment:
not a suggestion, but to understand better:

can CSSSelector be abstract so this asString() is abstract too?
It seems that in this case `toString()` wont' return anything meaningful. Is it right?

Reply:
anyway, what do you think about making CSSSelector abstract?
Can CSSSelector be instantiated on its own? If yes, It seems I don't know about use cases when it's needed. Could you please tell me?

Reply:
of course this question is not directly related to this PR. just asking

Reply:
it is good idea to make CSSSelector abastract